### DISCUSSION TITLE
Authentication fails on login

### DISCUSSION THREAD

**Discussion by chieflewus** [outline/outline#6916] [Self Hosting]
I've got a fresh install of the latest version of Outline. I'm attempting to use Authentik and Slack for authentication and I keep getting the same error over and over. I get a "Authentication failed. We were unable to sign you in at this time. Please try again later" I also get that last image in my URL on the failed login. Not sure if that is of any relevance or not.

Everything seems to be correct. I've of course redacted my secret keys and such but they are as provided by Slack and Authentik. I've never got to login either. Second fresh install and same results.

Info from my .env file
![authentik1](https://github.com/outline/outline/assets/88015106/4469e4b2-96d8-4c28-b3bd-c0b560d3b64a)

Authentik setup
![authentik2](https://github.com/outline/outline/assets/88015106/a0d05ba5-2750-462a-8909-c1b82580fc81)

Login page showing both Slack and Authentik
![authentik3](https://github.com/outline/outline/assets/88015106/6f776c02-089c-4dbc-9206-b1ad051c5b95)

Error received from either application
![authentik4](https://github.com/outline/outline/assets/88015106/1d06e4cd-d8ab-47bd-a21e-bba83665fcc9)

URL I receive on failure
![authentik5](https://github.com/outline/outline/assets/88015106/9c4ec156-541e-425b-8b29-86fd0b28941d)

**Comment by tommoor** [comment]
`state-mismatch` could mean a number of things, last time It came up it was because the server time was wrong – you could check that.

https://github.com/outline/outline/discussions/6294
